doc/changelog/01-kernel/19671-master+fix19661-guard-with-uniform-parameters.rst

```diff
@@ -0,0 +1,5 @@
+- **Fixed:**
+  guard checker regression in the presence of nested beta-redexes across :g:`match`
```

Comment:
```suggestion
  Guard checker anomaly in the presence of nested beta-redexes across :g:`match`.
  This is a regression in 8.20
```

Too bad we don't say much about guard checking in the doc.  The term is mentioned but nowhere discussed in detail AFAICT.

Reply:
I reworded on my side and included your suggestion.